guides/common/modules/con_red-hat-helper-app.adoc

```diff
@@ -1,5 +1,7 @@
 :_mod-docs-content-type: CONCEPT
 
+[id="red-hat-helper-app_{context}"]
```

Comment:
```suggestion
[id="red-hat-satellite-helper-app"]
```
* Adding `_{context}` is no longer needed
* The `{ProjectName}` in section heading actually translates to `Red Hat Satellite`